cdap-app-fabric/src/main/java/io/cdap/cdap/internal/provision/ProvisioningService.java

```diff
@@ -426,7 +436,7 @@ public Optional<RuntimeJobManager> getRuntimeJobManager(ProgramRunId programRunI
     String user = programOptions.getArguments().getOption(ProgramOptionConstants.USER_ID);
     Map<String, String> properties = SystemArguments.getProfileProperties(systemArgs);
     ProvisionerContext context = createContext(cConf, programOptions, programRunId, user,
-        properties, null);
+        properties, null, new ErrorCategory(ErrorCategoryEnum.STARTING));
```

Comment:
getRuntimeJobManager is called from getClusterStatus, so it applies here as well

Reply:
I thought `getRuntimeJobManager` is used while launching the `dataproc job`. Currently, I am not sure what to pass as error category from `getClusterStatus` as well.

So marked it as `null` for now, will change later after some more deeper look in follow up PR. If you have any suggestions, let me know.